Geometry/HGCalMapping/plugins/alpaka/HGCalMappingModuleESProducer.cc

```diff
@@ -108,6 +109,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
     private:
       edm::ESGetToken<HGCalMappingModuleIndexer, HGCalElectronicsMappingRcd> moduleIndexTkn_;
       const edm::FileInPath filename_;
+      const int nCEELayers_;
```

Comment:
Do you foresee that this may become configurable or geometry-dependent in the future ?

If _not_, it could be

```suggestion
      static constexpr int nCEELayers_ = 26;
```